contrib/natsio/README.md

```diff
@@ -0,0 +1,23 @@
+# natsio
+
+Google Cloud Dataflow Java Connector for [NATS messaging system](http://nats.io).
+
+## Getting Started
+
+Clone Dataflow SDK with natsio Connector and built them with Maven.
+
+Install and launch [NATS server](http://nats.io/documentation/server/gnatsd-intro/), typically on Google Compute Engine.
+```bash
+% gnatsd
+```
+
+Run tests with Maven. The following command will launch a producer and consumer job respectively.<BR>
+The producer publishes messages and the consumer will receive them via NATS server.
+
+"stagingLocation", "project" and "nats.servers" are mandatory properties to run a test.
+```bash
+% mvn test -Dtest=NatsIOTest#publishSubscribe -DstagingLocation=gs://<bucket> ¥
+-Dproject=<project id> -Dnats.servers=nats://<server>:4222 -Dnats.queue=queue1 ¥
+-Dloop=30000 -Dinterval=0 -Dsubjects=test1 -Dconsumers=1 -Dproducers=1 -Dnats.maxRecords=20000 ¥
```

Comment:
This symbol that is rendering as a Yen sign for me -- is this a Japanese keyboard version of `\` to mean "continues on next line"?